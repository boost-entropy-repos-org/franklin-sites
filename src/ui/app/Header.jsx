import React from 'react';
import {
  Link,
} from 'react-router-dom';

const Header = () => (
  <div className="header">
    <h3>Franklin</h3>
    <ul className="">
      <li>
        <Link to="/">Home</Link>{' '}
      </li>
      <li>
        <Link to="/ui-components">UI components</Link>{' '}
      </li>
    </ul>
  </div>
);

export default Header;

