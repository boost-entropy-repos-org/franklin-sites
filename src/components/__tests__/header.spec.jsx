import React from 'react';

import Header from '../header';
import renderWithRouter from '../../testHelpers/renderWithRouter';

describe('Header component', () => {
  test('should render', () => {
    const { asFragment } = renderWithRouter(
      <Header links={[{ path: '/there', label: 'there' }]} />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
