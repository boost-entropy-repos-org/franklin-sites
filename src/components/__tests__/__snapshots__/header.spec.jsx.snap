// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="header__logo"
    >
      <a
        href="/"
      />
    </div>
    <ul
      class="header__navigation"
    >
      <li>
        <a
          href="/there"
        >
          there
        </a>
      </li>
    </ul>
    <div
      class="header__search"
    />
  </div>
</DocumentFragment>
`;
